Sources/SingleFactorAuth/SessionData.swift

```diff
@@ -2,32 +2,26 @@ import BigInt
 import Foundation
 import TorusUtils
 
-public class SessionData: Codable {
-    let privateKey: String
-    let publicAddress: String
-    let signatures: TorusKey.SessionData?
-    let userInfo: UserInfo?
+public struct SessionData: Codable {
+    public let privateKey: String
+    public let publicAddress: String
+    public let signatures: [String]?
+    public let userInfo: UserInfo?
+    public let sessionNamespace: String
```

Comment:
This should be private and not be accessible? It's for internal use only, and it can be confusing with SessionNamespace of EIP155 or Solana.

Reply:
done.